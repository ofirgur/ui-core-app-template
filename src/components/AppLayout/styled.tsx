import styled from '@emotion/styled';
import { blue } from '@mui/material/colors';

export const AppLayoutStyled = styled.div`
  min-height: 100vh;
  display: flex;
  background: ${blue[500]};
  color: #fff;
`;
