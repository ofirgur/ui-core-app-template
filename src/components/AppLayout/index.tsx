import React from 'react';
//import { useTheme } from 'styled-components';
import LeftPanel from './components/LeftPanel';
import Main from './components/Main';
import { AppLayoutStyled } from '../AppLayout/styled';

const AppLayout = () => {
  //const theme = useTheme();
  //console.log('theme: ', theme);
  return (
    <AppLayoutStyled>
      <LeftPanel />
      <Main />
    </AppLayoutStyled>
  );
};

export default AppLayout;
