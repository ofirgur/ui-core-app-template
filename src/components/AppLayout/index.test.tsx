import * as React from 'react';
import { render } from '@testing-library/react';
import AppLayout from '.';

describe('App', () => {
  it('should render correctly', () => {
    const { container } = render(<AppLayout />);
    expect(container).toMatchSnapshot();
  });
});
