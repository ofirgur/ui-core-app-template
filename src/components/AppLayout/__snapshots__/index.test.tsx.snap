// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLayout should render correctly 1`] = `
<div>
  <div
    class="sc-eCYdqJ bOVqNy"
  >
    <div
      class="sc-bczRLJ hDKnho"
    >
      Left Panel
    </div>
    <main
      class="sc-hKMtZM FQBQW"
    >
      <h2
        class="sc-gsnTZi XESAr"
      >
        Header
      </h2>
      <div
        class="sc-dkzDqf wMITy"
      />
    </main>
  </div>
</div>
`;
